cmd/module/main.go

```diff
@@ -2,12 +2,14 @@ package main
 
 import (
 	"obstaclespointcloud"
+
 	"go.viam.com/rdk/module"
 	"go.viam.com/rdk/resource"
 	"go.viam.com/rdk/services/vision"
 )
 
 func main() {
 	// ModularMain can take multiple APIModel arguments, if your module implements multiple models.
-	module.ModularMain(resource.APIModel{ vision.API, obstaclespointcloud.ObstaclesPointcloud})
+	module.ModularMain(resource.APIModel{API: vision.API, Model: obstaclespointcloud.ObstaclesPointCloud},
+		resource.APIModel{API: vision.API, Model: obstaclespointcloud.ObstaclesDepth})
```

Comment:
```suggestion
	module.ModularMain(
		resource.APIModel{API: vision.API, Model: obstaclespointcloud.ObstaclesPointCloud},
		resource.APIModel{API: vision.API, Model: obstaclespointcloud.ObstaclesDepth})
```
That way, it's easier to see that there are two very similar APIs provided.